apis/Google.Cloud.PubSub.V1/Google.Cloud.PubSub.V1.Tests/Google.Cloud.PubSub.V1.Tests.csproj

```diff
@@ -40,6 +40,10 @@
     <Folder Include="Tasks\" />
   </ItemGroup>
 
+  <ItemGroup>
```

Comment:
I'm going to add this to the project generator, to stop it from coming up on random reviews...